include/watchdog.h

```diff
@@ -28,37 +28,6 @@
 extern "C" {
 #endif
 
-/**
```

Comment:
I've seen a few commits that argue "unused" they can be deleted, but unused in-tree doesn't mean unused out of tree, and we have an API deprecation policy now.

Does this patch follow it?